src/components/settings/cameras/Cameras.vue

```diff
@@ -49,9 +49,19 @@
             text
             x-small
             color=""
+            :disabled="camera.source === 'config'"
             @click.stop="handleRemoveCamera(camera)"
           >
-            <v-icon color="">
+            <v-icon
```

Comment:
Not sure if a title prop would be appropriate here explaining why the config is locked (maybe moving `app.timelapse.tooltip.managed_by_moonraker` to `app.general.tooltip.managed_by_moonraker` and use that?)
LMK if you want me to push this

Reply:
Good point, I'll change that!

Reply:
Done!